pydantic_ai_slim/pyproject.toml

```diff
@@ -72,6 +72,7 @@ bedrock = ["boto3>=1.37.24"]
 # Tools
 duckduckgo = ["duckduckgo-search>=7.0.0"]
 tavily = ["tavily-python>=0.5.0"]
+aci = ["aci-sdk>=1.0.0b2"]
```

Comment:
CI is failing because `aci-sdk` requires Python 3.10 and we still support Python 3.9 :( Can you please remove this? And add back the `try/except` import stuff you mentioned in a comment, but change the recommendation to install `aci-sdk` directly instead of using the dependency group.